software/controller/test/uart/uart_test.cpp

```diff
@@ -47,20 +47,25 @@ TEST(UartTest, Tx) {
   // create a buffer slightly bigger than the tx buffer
   size_t buffer_size{test_uart.GetBufferLength() + 1};
   uint8_t send_buffer[buffer_size];
-  for (uint8_t i = 0; i < buffer_size; ++i) send_buffer[i] = i;
+  for (size_t i = 0; i < buffer_size; ++i) send_buffer[i] = static_cast<uint8_t>(i);
```

Comment:
To me the flow is more readable when broken across two lines, and more safe with added {}